src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -216,6 +218,14 @@ ChipError::StorageType pychip_DeviceController_StackInit()
 
     FactoryInitParams factoryParams;
     factoryParams.fabricIndependentStorage = sStorageAdapter;
+
+    // TODO: Provide a means to Python-configure the group data provider to update local IPK
+    sGroupDataProvider.SetStorageDelegate(&sStorageAdapter);
+    ReturnErrorOnFailure(sGroupDataProvider.Init().AsInteger());
+
+    // TODO: Set IPK from src/lib/support/TestGroupData.h:DefaultIpkValue::GetDefaultIpk()
+
+    factoryParams.groupDataProvider        = sGroupDataProvider;
```

Comment:
@erjiaqing Can you please see if you can properly init this file and `src/controller/python/chip/internal/CommissionerImpl.cpp` to have the IPK (see https://github.com/project-chip/connectedhomeip/pull/16737/files#diff-7d33bc2a1145c23e5f424c29d4377e9dd9845f4db0b0ebdfa34286647b39f831R126)